user-api/src/main/kotlin/com/sns/user/core/config/SwaggerTag.kt

```diff
@@ -0,0 +1,7 @@
+package com.sns.user.core.config
+
+class SwaggerTag {
+    companion object {
+        const val SIGN_UP: String = "SIGN_UP"
+    }
+}
```

Comment:
```suggestion
object SwaggerTag {
    const val SIGN_UP: String = "SIGN_UP"
}
```

한 단계 감싸고 싶으시면 이게 더 나을거에요